run.sh

```diff
@@ -51,24 +51,25 @@ main() {
   # Download WordPress
   # ------------------
   if [ ! -f /app/wp-settings.php ]; then
-    h3 "Downloading wordpress"
+    h2 "Downloading wordpress"
     chown -R www-data:www-data /app /var/www/html
     WP core download |& loglevel
+    h3 "Wordpress downloaded"
```

Comment:
I'd reword these two to something along the lines of `Initializing..` and `Downloading WordPress` simply because the `STATUS "${PIPESTATUS[0]}"` can potentially show a pass or a fail. "WordPress downloaded" to me denotes it will always pass.